src/index.html

```diff
@@ -31,7 +31,15 @@
     <script nomodule src="/build/manifold-subscription.js"></script>
   </head>
   <body>
+    <!--
+    <manifold-init
+      auth-token="eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiIyMWoyM3ZrdDZwY3A1eDg0NjRlZXZ4ZTI4NGg4ZyIsInN1YiI6ImJyYW4iLCJpYXQiOiIyMDIwLTA0LTE1VDE2OjI1OjIxLjM2Nzg5MDgxWiIsImV4cCI6IjIwMjAtMDQtMTZUMTY6MjU6MjEuMzY3ODkwODFaIiwiYXVkIjoibWFuaWZvbGQuY28vZ2F0ZWtlZXBlciIsInZhbHVlIjoiTDVYajgtOFMiLCJwbGF0Zm9ybSI6IjN6eDNxZWIxNXo3MGtjdjRuaDhjamVkaG5kd3oyIiwidG9rZW5fc2VjcmV0IjoiaDVxcDd1eTdMTnA0MTR2bjJTMUxnYzFuVm1XOEJuRGxvQXpiY0Nvb05wYVNuQ2oxT3BRZW1jUUphQmNCIn0.77WgsDjSnOFYTTASofYewNzsdNPNO9hjbqZuWXGFIs0zVj3XxVceUUAzD5WQcyl-fjD47mp9E8YN5E3N15XA0g"
+      env="stage"
+      client-id="3zx3qeb15z70kcv4nh8cjedhndwz2"
+    ></manifold-init>
+    -->
```

Comment:
Why is this commented out?

Reply:
Oh I don't know, probably just a bad way of dealing with the fact that the component should have an auth token but that it can't be static. I can remove it.